src/Workspaces/Remote/Core/Services/AssetStorage.cs

```diff
@@ -18,28 +18,37 @@ namespace Microsoft.CodeAnalysis.Remote
     internal class AssetStorage
     {
         // TODO: think of a way to use roslyn option service in OOP
-        public static readonly AssetStorage Default = new AssetStorage(cleanupInterval: TimeSpan.FromMinutes(1), purgeAfter: TimeSpan.FromMinutes(3));
+        public static readonly AssetStorage Default =
+            new AssetStorage(cleanupInterval: TimeSpan.FromMinutes(1), purgeAfter: TimeSpan.FromMinutes(3), gcAfter: TimeSpan.FromMinutes(5));
```

Comment:
:question: Why are these values changing as part of this pull request?

Reply:
it is not changing, it is adding new option called gcAfter. 64bit require more explicit clean up than 32bit since it can leave a lot bigger unused memory space around. lessen learned from VBCSCompiler. this make sure if there was no activity on OOP for 5 mins, I will call explicit GC cleanup.

Reply:
:bulb: This should be added as a comment on the `_gcAfterTimeSpan` field and/or the documentation for the `gcAfter` constructor parameter.

Reply:
sure